keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -74,6 +75,7 @@ class ArtifactController(
     path = ["/pin"]
   )
   @ResponseStatus(ACCEPTED)
+  @PreAuthorize("@authorizationSupport.hasApplicationPermission('WRITE', 'DELIVERY_CONFIG', #pin.deliveryConfigName)")
```

Comment:
It looks like pin and delete pin are different permissions. Is that intentional?

Reply:
No, good spot. Will fix.